### DISCUSSION TITLE
Side by Side Translating

### DISCUSSION THREAD

**Discussion by midlos** [immersive-translate/immersive-translate#1900] [Ideas]
### 新功能 | New Feature

There's an awesome option named "Show Bilingual Translation" that you can use when you come across articles and texts. It would be amazing if it would be possible to, in addition to showing the original just below, have an additional option to show it side by side.

### 为什么您想要添加这些功能 | Why you want to add these features

It's easier to the reader when the translation is side by side. For example, for comparing and proofreading and also for language learning and academic research.

**Comment by theowenyoung** [comment]
I'm sorry, but our current technology isn't quite up to it.
